src/libraries/System.Numerics.Tensors/tests/TensorPrimitives.Generic.cs

```diff
@@ -371,18 +358,27 @@ public static IEnumerable<object[]> SpanDestinationFunctionsToTest()
             yield return new object[] { new SpanDestinationDelegate(TensorPrimitives.Atanh), new Func<T, T>(T.Atanh) };
             yield return new object[] { new SpanDestinationDelegate(TensorPrimitives.AtanPi), new Func<T, T>(T.AtanPi) };
             yield return new object[] { new SpanDestinationDelegate(TensorPrimitives.Atan), new Func<T, T>(T.Atan) };
-            yield return new object[] { new SpanDestinationDelegate(TensorPrimitives.Cbrt), new Func<T, T>(T.Cbrt) };
+            // TODO https://github.com/dotnet/runtime/issues/98861
```

Comment:
Is the idea with these TODOs that subsequent to this PR we'll burn down through all the TODOs, either accepting the tolerance or improving the implementation, and then either way removing the TODO?

Reply:
Correct. Ideally we should pick a baseline tolerance that works for this library and then improve every implementation that deviates.